### ISSUE TITLE
Bootstrap carousel controls not properly clickable in IE9

### ISSUE THREAD

**Issue by Cellcote** [twbs/bootstrap#16796]
So I have included a Bootstrap carousel in my website and I'm creating my own controls for going to the next slide. When I tried to check if the controls were working in IE9, I found something strange. The clickable area for the control was gone. I checked the default controls if the same thing occurred, and it did. To be more concrete, I made some screenshots.

In Safari, Chrome and IE11, this is what's happening and what is desired: ![screenshot](http://i.imgur.com/lULnWub.jpg)
Here the red area is the area in which the click will trigger the next slide. The left side is my own control, right side is the default glyphicon.

In IE9, this will happen: ![screenshot](http://i.imgur.com/qjfrp1o.png)

Is there a quick fix for this?

**Comment by cvrebert** [comment]
@Cellcote Please confirm that you're using Bootstrap v3.3.5.
X-Ref: #11186

**Comment by Cellcote** [comment]
I just upgraded to Bootstrap v3.3.5 and this issue is still happening.

**Comment by cvrebert** [comment]
@Cellcote Please post a JS Bin or JS Fiddle that demonstrates the problem.

**Comment by cvrebert** [comment]
Nevermind, confirmed using http://getbootstrap.com/examples/carousel/

**Comment by Cellcote** [comment]
A jsfiddle is [here](https://jsfiddle.net/DTcHh/9789/)

Inspired by the issue linked, i managed to fix this by adding the following css:
```
a.carousel-control {
        background-color: rgba(0,0,0,0);
}
```
This solution works in IE9, but does not work in IE8

**Comment by twbs-lmvtfy** [comment]
Hi @Cellcote!

You appear to have posted a live example (https://fiddle.jshell.net/DTcHh/9789/show/light/), which is always a good first step. However, according to [Bootlint](https://github.com/twbs/bootlint), **your example has some Bootstrap usage errors**, which might potentially be causing your issue:
* line 15, column 7: [W013](https://github.com/twbs/bootlint/wiki/W013): Bootstrap version might be outdated. Latest version is at least 3.3.4 ; saw what appears to be usage of Bootstrap 3.0.0

You'll need to **fix these errors** and post a revised example before we can proceed further.
Thanks!

(*Please note that this is a [fully automated](https://github.com/cvrebert/lmvtfy) comment.*)

**Comment by cvrebert** [comment]
Relevant docs: https://developer.mozilla.org/en-US/docs/Web/Events/click#Internet_Explorer

**Comment by mdo** [comment]
We can fix the IE9 part, but calling IE8 a won't fix since the suggested fixes in the linked MDN document for IE8 reference nuking the opacity as the only way. That kind of defeats the point though—if it's invisible, there's literally nothing to click on :grin:.
